pep-0678.rst

```diff
@@ -215,6 +228,9 @@ exception chaining, and are unnecessary with ``BaseException.__note__`` :
     Explanation:                                                            # Hence this PEP!
     You can reproduce this error by ...
 
+**Where these factors are not problematic, we encourage use of exception chaining
```

Comment:
What does "problematic" mean here?

Reply:
Its referring to the two factors mentioned in the two paragraphs above the two code blocks, which are admittedly a bit distant (and could justify, for example, something more like

```suggestion
**Where the two factors mentioned above are not problematic, we encourage use of exception chaining
```

to make it more clear.

Reply:
I wasn't very clear -- my comment was more that "problematic" as a word is quite vague (and not very common) -- I'd suggest something like
```suggestion
**Where these two problems are not relevant, we encourage using `exception chaining`_.**
```
This uses the "problems" phrasing from the lead paragraph, and uses more explicit / plainer English.

A

Reply:
I hear "problematic" used all the time in AmE, so maybe its an AmE-BrE distinction (though I'd think its meaning would seem to be pretty clear, so long as readers are familiar with the root word and the suffix).

Reply:
Fair enough. Not major really. (This might also be a circles one moves in thing? Anyhow)
A

Reply:
I think its a fair point to make; if it isn't going to be understandable to a large fraction of readers especially internationally, then it should be revised, and assuming we want to do so, your edit is a good way to accomplish that (and also adds the link and clarifies the reference somewhat)